library/alloc/src/alloc.rs

```diff
@@ -342,3 +362,30 @@ pub fn handle_alloc_error(layout: Layout) -> ! {
     }
     unsafe { oom_impl(layout) }
 }
+
+#[cfg(not(any(test, bootstrap)))]
+#[doc(hidden)]
+#[allow(unused_attributes)]
+#[unstable(feature = "alloc_internals", issue = "none")]
+pub mod __default_lib_allocator {
+    use crate::alloc::Layout;
+
+    // called via generated `__rust_alloc_error_handler`
+
+    // if there is no `#[alloc_error_handler]`
+    #[rustc_std_internal_symbol]
+    pub unsafe extern "C" fn __rdl_oom(_size: usize, _align: usize) -> ! {
+        panic!("memory allocation failed")
+    }
+
+    // if there is a `#[alloc_error_handler]`
+    #[rustc_std_internal_symbol]
+    pub unsafe extern "C" fn __rg_oom(size: usize, align: usize) -> ! {
+        let layout = Layout::from_size_align(size, align).unwrap();
```

Comment:
Use `from_size_align_unchecked` here to avoid the `unwrap`.

Reply:
yeah, figured that 3 minutes ago :) ... compiling :)

Reply:
That fixed the issue :smile: